Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+
+The primary objective of the evaluation process is to establish a level of confidence that the security functionalities of these DApps and the assurance measures align with the predefined requirements. Consequently, the evaluation results serve as a valuable resource for end-users seeking to gauge whether a particular DApp aligns with their security requirements and expectations.
+
+The Cardano Certification Standard also serves as a guide for both DApp developers and evaluators. It offers insights and best practices for the development of DApps with robust security features and provides a structured approach for evaluating their security posture.
+
+Flexibility is a core principle of the Cardano Certification Standard. It is designed to accommodate a diverse range of evaluation methodologies, ensuring that all essential security properties required by a DApp can be effectively assessed. However, it is essential to note that the validity of an evaluation is contingent upon the specific security properties evaluated and the methods employed. Auditors and users are advised to examine the DApp's security requirements and evaluation methodologies to determine the meaningfulness of the evaluation results.
+
+The Cardano Certification Standard aims at adressing:
+
+- The protection of assets of any user interacting with the DApp
+- The loss of use of the DApp
+- ...
+
+Moreover, the scope of the Cardano Certification Standard could extend to security aspects beyond these categories.
+
+Nevertheless, it is important to acknowledge that certain topics are explicitly excluded from the purview of the Cardano Certification Standard.
+
+Among these exclusions are:
+- ...
+
+In conclusion, the Cardano Certification Standard serves as a document promoting transparency, consistency, and confidence in the security of DApps.
+
+*RSoulat* This document draws heavily from the work of Common Criteria. I think we should take some inspiration from it for the security aspects, and, if we want to deal with purely functional aspects, add them where we see fit.
+
+## 2 General model
+
+### 2.1 Scope
+
+The CCS wishes to establish the general concepts and principles of DApp security evaluation and to specify the general model of evauation. This section provides an overview of all the parts of the CCS, it defines all the different terms, abbreviations and concepts used through the rest of the document. It establishes the core principles of the standard:
+
+- Protection Profiles
+- Security Targets
+- Target of Evaluation
+- Evaluation Levels
+
+### 2.2 Terms and Definitions
+
+*RSoulat* Note: Taken from the CC definitions
+*RSoulat* Maybe we could drop the Protection Profiles from a first version and only deal with Security Target
```

Comment:
I'd encourage taking a minimalist approach: only include things if it's really necessary. In particular, something like profiles might just be a reference back to the common criteria standard for information, for example.

Reply:
I agree. 

The Protection Profiles are interesting but I think they will come later in the use of the standard. For example, a DEX might come in the future with a more standardized set of security requirements that needs to be verified and this could become a Profile.